core/src/sumeragi/fault.rs

```diff
@@ -267,7 +276,7 @@ impl<F: FaultInjection> SumeragiWithFault<F> {
 
     pub fn wait_for_peers(&self, network_topology: &Topology) {
         iroha_logger::info!("Waiting for active peers",);
-        for i in 0..80 {
+        for i in 0..120 {
```

Comment:
Explain, please.

Reply:
Same as above. You want the test to fail in a reasonable time but wait long enough so that it always passes if the code works.